### ISSUE TITLE
Feature suggestions

### ISSUE THREAD

**Issue by SBPrime** [SBPrime/MidiPlayer#4]
Feedback messages so you know if its supposed to be playing r what
Option to enable simpler command aliases (/p, /pg)
Play MIDI to specified persons
Play MIDI at specified coordinates
Have the midi files in a MIDI folder instead of in the plugin's folder
Allow referring to MIDIs in external directories
Allow playing MIDIs from URL and/or uploading to server
Index MIDIs so you can...
List available MIDIs
Tab complete and search MIDIs
Play random MIDI

Force midi to play with only a specified instrument
Adjust octave & transpose
Playlists
Repeat
Basically all the good stuff a MIDI play would have
